ORStools/proc/isochrones_point_proc.py

```diff
@@ -27,166 +27,87 @@
-
-        self.addParameter(
-            QgsProcessingParameterFeatureSink(
-                name=self.OUT,
-                description="Isochrones",
-                createByDefault=False
-            )
-        )
-
-    def group(self):
-        return "Isochrones"
-
-    def groupId(self):
-        return 'isochrones'
-
-    def name(self):
-        return self.ALGO_NAME
-
-    def shortHelpString(self):
-        """Displays the sidebar help in the algorithm window"""
-
-        file = os.path.join(
-            HELP_DIR,
-            'algorithm_isochrone_point.help'
-        )
-        with open(file, encoding='utf-8') as helpf:
-            msg = helpf.read()
-
-        return msg
-
-    def helpUrl(self):
-        """will be connected to the Help button in the Algorithm window"""
-        return __help__
+        ]
 
-    def displayName(self):
-        return " ".join(map(lambda x: x.capitalize(), self.ALGO_NAME_LIST))
-
-    def icon(self):
-        return QIcon(RESOURCE_PREFIX + 'icon_isochrones.png')
-
-    def createInstance(self):
-        return ORSisochronesPointAlgo()
+    # Save some important references
+    # TODO bad style, refactor
+    isochrones = isochrones_core.Isochrones()
+    dest_id = None
+    crs_out = QgsCoordinateReferenceSystem.fromEpsgId(4326)
+    # difference = None
 
     # TODO: preprocess parameters to options the range cleanup below:
     # https://www.qgis.org/pyqgis/master/core/Processing/QgsProcessingAlgorithm.html#qgis.core.QgsProcessingAlgorithm.preprocessParameters
-
     def processAlgorithm(self, parameters, context, feedback):
-        # Init ORS client
-        providers = configmanager.read_config()['providers']
-        provider = providers[self.parameterAsEnum(parameters, self.IN_PROVIDER, context)]
-        clnt = client.Client(provider)
-        clnt.overQueryLimit.connect(lambda: feedback.reportError("OverQueryLimit: Retrying..."))
-
-        params = dict()
-        params['attributes'] = ['total_pop']
+        ors_client = self._get_ors_client_from_provider(parameters[self.IN_PROVIDER], feedback)
 
-        profile = PROFILES[self.parameterAsEnum(parameters, self.IN_PROFILE, context)]
-        params['range_type'] = dimension = DIMENSIONS[self.parameterAsEnum(parameters, self.IN_METRIC, context)]
+        profile = dict(enumerate(PROFILES))[parameters[self.IN_PROFILE]]
+        dimension = dict(enumerate(DIMENSIONS))[parameters[self.IN_METRIC]]
 
-        factor = 60 if params['range_type'] == 'time' else 1
-        ranges_raw = self.parameterAsString(parameters, self.IN_RANGES, context)
+        factor = 60 if dimension == 'time' else 1
+        ranges_raw = parameters[self.IN_RANGES]
         ranges_proc = [x * factor for x in map(int, ranges_raw.split(','))]
-        params['range'] = ranges_proc
 
         point = self.parameterAsPoint(parameters, self.IN_POINT, context, self.crs_out)
 
         # Make the actual requests
         # If layer source is set
-        requests = []
```

Comment:
… did that just do nothing before, or am I reading that wrong?

Reply:
exactly, it was a copy paste remainder